proposals/artifacts.md

```diff
@@ -0,0 +1,78 @@
+# OCI Artifact Project Proposal
```

Comment:
s/Artifact/Artifacts/
||
s/Artifact/Artifact Types/

Reply:
Going with Artifacts, as it seems a it more broad and inclusive. Artifact Types sounds like a repo specific to the individual types.